data/Localization/german_(germany)/global.ini

```diff
@@ -60891,7 +60891,7 @@ missionManager_CrusaderHunted_Objective_03_Long=Ihr CrimeStat-Rating senken oder
 missionManager_CrusaderHunted_Objective_03_Short=Überlebensgarantie
 missionManager_CrusaderHunted_Title=Tipps und Tricks zum Überleben Ihres Todesurteils
 missionManager_CrusaderHunted_Title_Short=Todesurteil
-missionManager_CrusaderWantedTutorial_Description=Hey, \n\nEine Kriminalitätsstatistik zu bekommen, kann eine echte Qual sein, richtig? Ein kleiner Fehler in einer überwachten Zone und schon sind die Sicherheitsleute hinter dir her. \n\nNatürlich könntest du dich verstecken, bis die Hitze nachlässt, aber du hast Glück, denn ich kenne eine kleine Lösung, die so ziemlich jede Verbrecher-Situation verbessert.\n\n Alles, was du tun musst, ist zum Sicherheitsposten Kareah zu gehen und ein Terminal zu finden, das mit der imperialen Verbrecher-Datenbank verbunden ist. Es ist nicht gerade die einfachste Station, in die man hineingehen kann, seit die Sicherheitskräfte der Kreuzfahrer begonnen haben, sie als Beweismittellager zu nutzen, und überall wimmelt es von Polizisten, aber man kann es sich ja nicht aussuchen, oder? \n\nSchnappen Sie sich einfach einen Kryptoschlüssel (Technotics in Grim HEX kann Ihnen vielleicht helfen), schleichen oder kämpfen Sie sich nach Kareah, hacken Sie die Verbrecherdatenbank, und schon gibt es keine Verbrecherquoten mehr. Ich bin mir sicher, dass sich das einfacher anhört, als es tatsächlich ist.\n \n Natürlich können Sie stattdessen auch gerne einen Entzug machen.\n\n-Ruto\n
+missionManager_CrusaderWantedTutorial_Description=Hey, \n\nEine Kriminalitätsstatistik zu bekommen, kann eine echte Qual sein, richtig? Ein kleiner Fehler in einer überwachten Zone und schon sind die Sicherheitsleute hinter dir her. \n\nNatürlich könntest du dich verstecken, bis die Hitze nachlässt, aber du hast Glück, denn ich kenne eine kleine Lösung, die so ziemlich jede Verbrecher-Situation verbessert.\n\n Alles, was du tun musst, ist zum Sicherheitsposten Kareah zu gehen und ein Terminal zu finden, das mit der imperialen Verbrecher-Datenbank verbunden ist. Es ist nicht gerade die einfachste Station, in die man hineingehen kann, seit die Sicherheitskräfte Crusaders begonnen haben, sie als Beweismittellager zu nutzen, und überall wimmelt es von Polizisten, aber man kann es sich ja nicht aussuchen, oder? \n\nSchnappen Sie sich einfach einen Kryptoschlüssel (Technotics in Grim HEX kann Ihnen vielleicht helfen), schleichen oder kämpfen Sie sich nach Kareah, hacken Sie die Verbrecherdatenbank, und schon gibt es keine Verbrecherquoten mehr. Ich bin mir sicher, dass sich das einfacher anhört, als es tatsächlich ist.\n \n Natürlich können Sie stattdessen auch gerne einen Entzug machen.\n\n-Ruto\n
```

Comment:
"bis die Hitze nachlässt" klingt falsch. Wäre "bis die Luft wieder rein ist" oder ähnliches nicht besser?

"Es ist nicht gerade die einfachste Station, in die man hineingehen kann" klingt auch falsch, vielleicht wäre ein "Es ist nicht gerade die einfachste Station, in die man *einbrechen* kann" nicht besser, oder ähnliches?